apps/api/src/app/integrations/usecases/create-integration/create-integration.usecase.ts

```diff
@@ -149,7 +149,7 @@ export class CreateIntegration {
       const defaultName =
         providers.find((provider) => provider.id === command.providerId)?.displayName ?? providerIdCapitalized;
       const name = command.name ?? defaultName;
-      const identifier = command.identifier ?? `${slugify(name)}-${shortid.generate()}`;
+      const identifier = command.identifier ?? `${slugify(name)}-${nanoid()}`;
```

Comment:
```suggestion
      const identifier = command.identifier ?? `${slugify(name)}-${shortId()}`;
```

the default is [length of 21](https://github.com/ai/nanoid/blob/e0f79c916ed7996f6a1547e3af4005dfe903ee65/index.js#L72C24-L72C28) 
regardless please use shortId from `libs/application-generic/src/utils/generate-id.ts` so we will have the same alphabet on our identifiers.